library/modules/form/controls/scss/input/floating.scss

```diff
@@ -14,21 +16,23 @@ $transition-speed: 0.2s;
 		width: 100%;
 		font-size: 1rem;
 		box-sizing: border-box;
-
+		@each $placeholder in $placeholder-devices {
```

Comment:
To mantain a clear structure, I suggest to put the focus style instructions before the each loop. Besids, the $placeholder-devices can ignore the "placeholder" string and you can concatenate it directly in the loop.